src/App.js

```diff
@@ -20,9 +20,13 @@ const useStyles = makeStyles((theme) => ({
 }));
 
 const log = console.log; //eslint-disable-line no-unused-vars
-let msgArray = [];
+let consoleMessages = [];
+let errorMessages = [];
 console.log = function getMessages(message) {
-  msgArray.push(message);
+  consoleMessages.push(message);
+  if (message && message.startsWith('error')) {
```

Comment:
I think it might be worthwhile to also show warnings, since that aligns well with the summary given by the box at the end of SUSHI.

Reply:
Okay! I was thinking about including them, but couldn't decide if it was too much information to have immediately available or if it would be helpful. But I'd say an additional vote for helpful tips the scales!

Reply:
Included in 7383c11!